lib/main.js

```diff
@@ -161,42 +161,130 @@ ava.test.serial(`resin host OS update [${options.resinOSVersion} -> ${options.re
 })
 
 if (!process.env.CI) {
-  ava.test('experience: get a shell into a running application container using the Resin CLI', async (test) => {
+  ava.test('connectivity: Provision on Ethernet', async (test) => {
```

Comment:
Why do we have test cases that talk about provisioning? The whole idea is that provisioning variants are implicit, and that the test cases are provisioning-variant independent